tests/test_treasury_module.py

```diff
@@ -94,6 +105,179 @@ def test_bundle_creation_with_riskpool_wallet_not_set(
             {'from': bundleOwner})
 
 
+
+def test_bundle_creation_allowance_too_small(
+    instance: GifInstance,
+    owner: Account,
+    testCoin,
+    productOwner: Account,
+    oracleProvider: Account,
+    riskpoolKeeper: Account,
+    capitalOwner: Account,
+):  
+    applicationFilter = bytes(0)
+
+    (gifProduct, gifRiskpool) = getProductAndRiskpool(
+        instance,
+        owner,
+        testCoin,
+        productOwner,
+        oracleProvider,
+        riskpoolKeeper,
+        capitalOwner,
+        True
+    )
+
+    # prepare allowance too small for riskpool funding 
+    safetyFactor = 2
+    amount = 10000
+    testCoin.transfer(capitalOwner, safetyFactor * amount, {'from': owner})

```

Comment:
verify that capitalOwner is not the riskpoolWallet. please use the riskpoolKeeper as the investor for the test case, not the capitalOwner

Reply:
done